ansible-scylla-node/templates/cassandra-rackdc.properties.j2

```diff
@@ -14,7 +14,7 @@
 #
 
 {% if scylla_snitch == 'GossipingPropertyFileSnitch' %}
-dc={{ dc }}
+dc={{ dc + dc_suffix | default('') }}
```

Comment:
This is a somewhat unexpected syntax. I was expecting something like this:

```
dc={{ dc }}{{ dc_suffix | default('') }}
```

Did you test your version?